### ISSUE TITLE
Look into setting test server class before starting mini accumulo

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#4644]
I'd bet that there are some other tests, related to Scan Servers and/or External Compactions, that could use this new method instead of stopping the "normal" implementation and starting a different one for the test. It's probably worth a new ticket for this.

_Originally posted by @dlmarion in https://github.com/apache/accumulo/pull/4643#discussion_r1630150595_

**Comment by dlmarion** [comment]
> Probably depends on if the test uses the normal server class that was started before it starts a test one.

It probably only makes sense to use this new method when the test class *only* uses the non-standard implementation. If the test class starts and stops different types of implementations, then this likely can't replace that functionality.

**Comment by cshannon** [comment]
@keith-turner - I think this can be closed after #4811, unless you had anymore tests you wanted to change. I looked and the other tests did not seem to be able to easily use this
